db/migration/000002_cli_user.up.sql

```diff
@@ -0,0 +1,25 @@
+
+ALTER TABLE users ADD COLUMN created_by UUID;
+-- SQL Definition for environments
+
+CREATE TABLE environments (
+    Id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    cli_user UUID NOT NULL REFERENCES users(id),
+    name VARCHAR(255) NOT NULL,
+    createdAt TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    updatedAt TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    created_by UUID NOT NULL REFERENCES users(id)
+);
+-- SQL Definition for environment fields
+
+CREATE TABLE environment_fields (
+    id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    field_name VARCHAR(255) NOT NULL,
+    field_value TEXT NOT NULL,
```

Comment:
do we need a field_value if the value is tied to a parent field

Reply:
both field values are different one is encrypted with users pub key other is encyrpted with cli users pub key